src/api/shared/create-redis.ts

```diff
@@ -17,7 +20,13 @@ export function createRedisFactory(redisUrl?: string): () => IORedis.Redis {
   let redis: IORedis.Redis | undefined = undefined;
   return () => {
     if (!redis) {
-      redis = new IORedis(redisUrl);
+      const options: RedisOptions = {}
+      if (process.env.REDIS_TLS_CA_FILE && fs.existsSync(process.env.REDIS_TLS_CA_FILE)) {
+        options.tls = {
+          ca: fs.readFileSync(process.env.REDIS_TLS_CA_FILE, 'utf8')
```

Comment:
I used the sync methods, as using the fs.promises would make the whole fn async - which we probably don't want at this point.